tests/test_bgt.py

```diff
@@ -1,5 +1,108 @@
-from bgt import __version__
+# std
+import os
+from random import randint
+from pathlib import Path
+from json import load, dump, loads
+
+# vendor
+from moto import mock_dynamodb2
+import pytest
+from pytest import fixture
+import boto3
+from starlette.testclient import TestClient
+
+# local
+from app._version import __version__
+from app.schema import Game
+from app.utils import UniversalEncoder
+
+
+@fixture
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ['AWS_ACCESS_KEY_ID'] = 'testing'
```

Comment:
I realize we never had a chance to talk about "yield fixtures", so this is a good occasion to do that. A yield fixture is effectively a fixture that's able to run code before and after a test. When you use this fixture the way it's defined currently, it will set environment variables and those variable will remain set for the lifetime of the process. (This may or may not be all that harmful, but typically, you'll want to have your fixtures clean up after themselves.)

To create a yield fixture you just `yield` instead of `return` the fixture value. When the test has finished and is cleaning up, the code after the `yield` statement inside your fixture will begin to run. 

For this fixture, it might mean something like:
```
@fixture(autouse=True)
def aws_credentials():
  missing = object()
  previous = os.environ.get("AWS_ACCESS_KEY_ID", missing)
  os.environ["AWS_ACCESS_KEY_ID"] = "testing"
  try:
    yield
  finally:
    if previous is missing:
      del(os.environ["AWS_ACCESS_KEY_ID"])
    else:
      os.environ["AWS_ACCESS_KEY_ID"] = previous

def test_something_or_other():
  assert os.environ["AWS_ACCESS_KEY_ID"] == "testing"
```

That's the hand-rolled version, but you could also use the `monkeypatch` fixture, which I believe is included with pytest out-of-the-box:

```
@fixture(autouse=True)
def _env_patches(monkeypatch):
  monkeypatch.setitem(os.environ, "AWS_ACCESS_KEY_ID", "testing")
  monkeypatch.setitem(os.environ, "AWS_SECRET_ACCESS_KEY", "testing")
  …
```